test/cli_test.py

```diff
@@ -848,3 +849,74 @@ def test_queue_peek_len_clear(servicer, server_url_env, set_env_client):
     _run(["queue", "clear", name, "--all", "--yes"])
     assert _run(["queue", "len", name, "--total"]).stdout == "0\n"
     assert _run(["queue", "peek", name, "--partition", "alt"]).stdout == ""
+
+
+@pytest.mark.parametrize(
+    "command",
+    [
+        ["environment", "create", ".main"],
```

Comment:
Nit but the only thing varying across the parameterizations is the name so you don’t need to parameterize the full command. Not a huge thing but will give better test output.